src/test/java/seedu/medibook/storage/JsonAdaptedPatientTest.java

```diff
@@ -49,11 +52,17 @@
     private static final List<JsonAdaptedTag> VALID_TAGS = BENSON.getTags().stream()
             .map(JsonAdaptedTag::new)
             .collect(Collectors.toList());
+    // medical note list data of patient should not be loaded at point of program start-up, hence expected
+    // behaviour is that all these patients have an empty medical note list instead of the one initialised and loaded
+    // in TypicalPatients.java for other testing purposes
```

Comment:
May look better to properly punctuate this comment?

Reply:
Ok thanks I have fixed this